Extensions.sln

```diff
@@ -58,6 +58,9 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Configuration.NewtonsoftJson.Tests", "src\Configuration\Config.NewtonsoftJson\test\Microsoft.Extensions.Configuration.NewtonsoftJson.Tests.csproj", "{2FEE9416-E709-4EA7-82A3-4A34816F75FD}"
 EndProject
 Global
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
```

Comment:
This was auto generated. I do not know why. If someone has knowledge of why please let me know.